src/pages/index.js

```diff
@@ -1,88 +1,109 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { graphql } from 'gatsby';
-import { Helmet } from 'react-helmet';
+import HubspotForm from 'react-hubspot-form';
 import jsYAML from 'js-yaml';
+import $ from 'jquery';
+import GitHubButton from 'react-github-btn';
+
+import 'slick-carousel/slick/slick.less';
 
 import EasyLayout from '../components/EasyLayout';
 
 import ImgBirdOnBricks         from '../assets/images/bird-on-bricks.svg';
-import ImgDebugLocally         from '../assets/images/debug-locally.svg';
-import ImgInstantFeedback      from '../assets/images/instant-feedback.svg';
-import ImgRealisticEnvironment from '../assets/images/realistic-environment.svg';
-import ImgMinimalResources     from '../assets/images/minimal-resources.svg';
-import ImgYourTools            from '../assets/images/your-tools.svg';
 import ImgQuote                from '../assets/images/quote.svg';
 import ImgVizualaiLogo         from '../assets/images/vizualai-logo.jpg';
 
+import SvgDebugLocally         from '../assets/images/debug-locally.inline.svg';
+import SvgInstantFeedback      from '../assets/images/instant-feedback.inline.svg';
+import SvgRealisticEnvironment from '../assets/images/realistic-environment.inline.svg';
+import SvgMinimalResources     from '../assets/images/minimal-resources.inline.svg';
+import SvgYourTools            from '../assets/images/your-tools.inline.svg';
+
 export default function HomePage({ location, data }) {
   const variables = jsYAML.safeLoad(data.file.internal.content);
 
+  useEffect(() => {
+    $(document).ready(function() {
```

Comment:
Is jQuery the only way we have to achieve this functionality? I think we can have functions to handle the click events inside the component rather than bind the events to functions through jQuery. 

We could have state `const [activeTab, setActiveTab]  = useState()` that can store the selected tab and a click handler that updates the state

const tabClickHandler = (tabId) => {
   setActiveTab(tabId)
};


The main reason for trying this approach is to get rid of jquery which can add an unnecessary overhead on the page (jQuery adds ~95kb to the bundle size)